components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Settings.php

```diff
@@ -44,6 +44,12 @@ public function __construct() {
 		// add the same tab for taxonomies
 		add_filter( 'pods_admin_setup_edit_tabs_taxonomy', array( $this, 'tab' ), 11, 3 );
 
+		// add the same tab for comments
+		add_filter( 'pods_admin_setup_edit_tabs_comment', array( $this, 'tab' ), 11, 3 );
+
+		// add the same tab for users / authors.
+		add_filter( 'pods_admin_setup_edit_tabs_user', array( $this, 'tab' ), 11, 3 );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 60726afcd6205620363522b28f8db4d4</sub>